src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -560,3 +561,113 @@ def _patch(self, crd, crd_name, cr_name, func):
+            deleted = False
+            retries = 0
+            while not deleted and retries < 10:
+                api_response = self.batchV1_api.list_namespaced_job(self.rook_env.namespace,
+                                                                    label_selector="ident=%s" % operation_id,
+                                                                    timeout_seconds=10)
+                deleted = not api_response.items
+                if retries > 5:
+                    sleep(0.1)
+                ++retries
+            if retries == 10 and not deleted:
+                raise orchestrator.OrchestratorError(
+                    "Light <{}> in <{}:{}> cannot be executed. Cannot delete previous job <{}>".format(
+                            on, loc.host, loc.path or loc.dev, operation_id))
+
+            # create the job
+            api_response = self.batchV1_api.create_namespaced_job(self.rook_env.namespace, job)
+
+            # get the result
+            finished = False
+            while not finished:
+                api_response = self.batchV1_api.read_namespaced_job(operation_id,
+                                                                    self.rook_env.namespace)
+                finished = api_response.status.succeeded or api_response.status.failed
+                if finished:
+                    message = api_response.status.conditions[-1].message
+
+            # get the result of the lsmcli command
+            api_response=self.coreV1_api.list_namespaced_pod(self.rook_env.namespace,
+                                                             label_selector="ident=%s" % operation_id,
+                                                             timeout_seconds=10)
+            if api_response.items:
+                pod_name = api_response.items[-1].metadata.name
+                message = self.coreV1_api.read_namespaced_pod_log(pod_name,
+                                                                  self.rook_env.namespace)
+
+        except ApiException as e:
+            log.exception('K8s API failed. {}'.format(e))
+            raise
+
+        # Finally, delete the job.
+        # The job uses <ttl_seconds_after_finished>. This makes that the TTL controller delete automatically the job.
+        # This feature is in Alpha state, so extra explicit delete operations trying to delete the Job has been used strategically
+        try:
+            api_response = self.batchV1_api.delete_namespaced_job(operation_id,
+                                                                  self.rook_env.namespace,
+                                                                  propagation_policy="Background")
+        except ApiException as e:
+            if e.status != 404: # No problem if the job does not exist
+                raise
+
+        return message
+
+    def blink_light(self, ident_fault: str, on: bool, locs: List[orchestrator.DeviceLightLoc]) -> List[str]:
```

Comment:
```
2020-02-29T11:45:43.588 INFO:tasks.ceph.mgr.y.smithi061.stderr:2020-02-29T11:45:43.585+0000 7f5cb714aec0 -1 mgr[py] Module not found: 'rook'
2020-02-29T11:45:43.589 INFO:tasks.ceph.mgr.y.smithi061.stderr:2020-02-29T11:45:43.585+0000 7f5cb714aec0 -1 mgr[py] Traceback (most recent call last):
2020-02-29T11:45:43.589 INFO:tasks.ceph.mgr.y.smithi061.stderr:  File "/usr/share/ceph/mgr/rook/__init__.py", line 2, in <module>
2020-02-29T11:45:43.589 INFO:tasks.ceph.mgr.y.smithi061.stderr:    from .module import RookOrchestrator
2020-02-29T11:45:43.589 INFO:tasks.ceph.mgr.y.smithi061.stderr:  File "/usr/share/ceph/mgr/rook/module.py", line 33, in <module>
2020-02-29T11:45:43.590 INFO:tasks.ceph.mgr.y.smithi061.stderr:    from .rook_cluster import RookCluster
2020-02-29T11:45:43.590 INFO:tasks.ceph.mgr.y.smithi061.stderr:  File "/usr/share/ceph/mgr/rook/rook_cluster.py", line 183, in <module>
2020-02-29T11:45:43.590 INFO:tasks.ceph.mgr.y.smithi061.stderr:    class RookCluster(object):
2020-02-29T11:45:43.590 INFO:tasks.ceph.mgr.y.smithi061.stderr:  File "/usr/share/ceph/mgr/rook/rook_cluster.py", line 672, in RookCluster
2020-02-29T11:45:43.591 INFO:tasks.ceph.mgr.y.smithi061.stderr:    def blink_light(self, ident_fault: str, on: bool, locs: List[orchestrator.DeviceLightLoc]) -> List[str]:
2020-02-29T11:45:43.591 INFO:tasks.ceph.mgr.y.smithi061.stderr:NameError: name 'List' is not defined
2020-02-29T11:45:43.591 INFO:tasks.ceph.mgr.y.smithi061.stderr:
```

http://pulpito.ceph.com/kchai-2020-02-29_11:07:48-rados-wip-kefu-testing-2020-02-29-1522-distro-basic-smithi/4813422/

Reply:
Yeah. Thanks. 
It seems that the mypy embedded notation is not ok. 
Replaced by comments

Reply:
```python
from typing import List
```
might be missing?